packages/k2-alpine/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropsWithChildren } from 'react'
+import { SxProp } from 'dripsy'
+import { View } from '../Primitives'
+import { useTheme } from '../..'
+
+export const Card = ({
+  sx,
+  children,
+  shouldCenterAlign = false,
```

Comment:
I think centering should be the default behavior. no need to provide a prop for is. the sx prop should be good enough.